docs/micro-services/index.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Micro-services
+layout: default
+---
+
+# Getting started  with micro-services
+
+Micro-services are implemented using [Spring Boot](https://spring.io/projects/spring-boot) and [Spring Cloud](https://spring.io/projects/spring-cloud), packaged as [Docker](https://www.docker.com/) images using [JIB Maven plugin](https://github.com/GoogleContainerTools/jib/tree/master/jib-maven-plugin).
+
+## Building docker images
+
+Clone git repository.
+
+```bash
+git clone https://github.com/powsybl/powsybl-spring.git
+```
+
+Compile and build docker images.
+
+```bash
+mvn -Pdocker install
+```
+
+Docker images should be available in your local registry.
+
+```bash
+$ docker images
+powsybl/powsybl-server-discovery                 1.0.0-SNAPSHOT                   b4aa78912a87        49 years ago        163.7 MB
```

Comment:
The `49 years ago` is weird, but I have the same output on my laptop

Reply:
the reason for this is explained here https://github.com/GoogleContainerTools/jib/blob/master/docs/faq.md#why-is-my-image-created-48-years-ago